tests/test_config.py

```diff
@@ -159,6 +160,13 @@ def test_get_config_verbose(tmp_path, caplog, capsys, verbose, force_interactive
         config = get_config()
         assert config.get("general.verbose") == verbose
 
-    regex = re.compile(f"Loaded.*config from.*{config_path}")
+    # On Windows the path in config_path would make the regex invalid because
```

Comment:
the `re` module has built-in escape function, just
```
regex = re.compile(f"Loaded.*config from.*{re.escape(config_path)}")
```
should do the trick cross-platform

Reply:
Done.